vue/components/ui/radio-platforme/radio-platforme.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+    <div class="radio-group">
+        <label-platforme class="title" v-bind:text="labelTitle" v-if="labelTitle" />
+        <div class="radio-group-choices">
+            <div
+                class="radio-group-choice"
+                v-bind:class="{
+                    disabled: disabled || item.disabled,
+                    error: error || item.error,
+                    checked: item.value === value
+                }"
+                v-for="item in items"
+                v-bind:key="item.value"
+                v-on:click="onClick(item)"
+            >
+                <input type="radio" class="value" v-bind:id="item.value" />
+                <div class="circular-button" />
+                <label v-bind:for="item.value" class="label">
+                    {{ item.label ? item.label : item.value }}
+                </label>
+            </div>
+        </div>
+        <label-platforme
+            class="footer"
+            v-bind:size="'small'"
+            v-bind:text="labelFooter"
+            v-if="labelFooter"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.radio-group {
+    display: inline-block;
+}
+
+.radio-group > .title {
+    color: $pale-grey;
+    display: block;
+}
+
+.radio-group-choice {
+    cursor: pointer;
+    display: block;
+    line-height: 13px;
+    padding: 10px 6px 10px 6px;
+    user-select: none;
+}
+
+.radio-group-choice:hover {
+    background-color: #ebecf0;
+}
+
+.radio-group-choice:hover > .circular-button {
+    border-color: #c1c7d0;
+}
+
+.radio-group-choice:active {
+    background-color: #dde0e2;
+}
+
+.radio-group-choice.disabled > .label {
+    cursor: not-allowed;
+}
+
+.radio-group-choice > .value {
+    display: none;
+}
+
+.radio-group-choice.disabled {
+    cursor: not-allowed;
+}
+
+.radio-group-choice > .circular-button {
+    background-color: #fafbfc;
+    border: 2px solid #dfe1e5;
+    border-radius: 50%;
+    display: inline-block;
+    height: 2px;
+    padding: 3px 3px 3px 3px;
+    vertical-align: bottom;
+    width: 2px;
+}
+
+.radio-group-choice:active:not(.disabled) > .circular-button {
+    background-color: $dark;
+    border-color: #c3c9cf;
+    border-width: 4px;
+    padding: 1px 1px 1px 1px;
+}
+
+.radio-group-choice.checked > .circular-button {
+    background-color: #ffffff;
+    border-color: $dark;
+    border-width: 4px;
+    padding: 1px 1px 1px 1px;
+}
+
+.radio-group-choice.error > .circular-button {
+    background-color: #f4f5f7;
+    border: 2px solid $red;
+}
+
+.radio-group-choice.error.checked > .circular-button {
+    border: 4px solid $red;
+}
+
+.radio-group-choice.disabled > .circular-button {
+    background-color: #f4f5f7;

```

Comment:
Use named colors instead.

Reply:
$pale-grey was added to colors.scss and used as variable instead of #f6f7f9.
As for other 8 different variations of gray, there are no names for them in zeplin palette or variables.

<img width="235" alt="imagem" src="https://user-images.githubusercontent.com/24736423/70052763-e7d10100-15cb-11ea-8719-b2b802d85979.png">

Reply:
Jorge just added the naming of the colors: zpl://screen?sid=5ddcf0ac47a99910ed09c278&pid=5db732f9c02f1e8327288741

Does this help?